src/coreComponents/constitutive/solid/LinearElasticTransverseIsotropic.cpp

```diff
@@ -42,91 +40,128 @@ LinearElasticTransverseIsotropic::LinearElasticTransverseIsotropic( string const
   registerWrapper( viewKeyStruct::defaultYoungsModulusTransverse, &m_defaultYoungsModulusTransverse )->
     setApplyDefaultValue( -1 )->
     setInputFlag( InputFlags::OPTIONAL )->
-    setDescription( "Elastic Bulk Modulus Parameter" );
+    setDescription( "Elastic Transverse Young's Modulus Parameter" );
 
   registerWrapper( viewKeyStruct::defaultYoungsModulusAxial, &m_defaultYoungsModulusAxial )->
     setApplyDefaultValue( -1 )->
     setInputFlag( InputFlags::OPTIONAL )->
-    setDescription( "Elastic Shear Modulus Parameter" );
+    setDescription( "Elastic Axial Young's Modulus Parameter" );
 
-  registerWrapper( viewKeyStruct::defaultPoissonRatioTransverse, &m_defaultPoissonTransverse )->
+  registerWrapper( viewKeyStruct::defaultPoissonsRatioTransverse, &m_defaultPoissonsRatioTransverse )->
     setApplyDefaultValue( -1 )->
     setInputFlag( InputFlags::OPTIONAL )->
-    setDescription( "Elastic Shear Modulus Parameter" );
+    setDescription( "Elastic Transverse Poisson's ratio Parameter" );
 
-  registerWrapper( viewKeyStruct::defaultPoissonRatioAxialTransverse, &m_defaultPoissonAxialTransverse )->
+  registerWrapper( viewKeyStruct::defaultPoissonsRatioAxialTransverse, &m_defaultPoissonsRatioAxialTransverse )->
     setApplyDefaultValue( -1 )->
     setInputFlag( InputFlags::OPTIONAL )->
-    setDescription( "Elastic Shear Modulus Parameter" );
+    setDescription( "Elastic Axial-Transverse Poisson's ratio Parameter" );
 
   registerWrapper( viewKeyStruct::defaultShearModulusAxialTransverse, &m_defaultShearModulusAxialTransverse )->
     setApplyDefaultValue( -1 )->
     setInputFlag( InputFlags::OPTIONAL )->
-    setDescription( "Elastic Shear Modulus Parameter" );
+    setDescription( "Elastic Axial-Transverse (out-plane) Shear Modulus Parameter" );
+
+  registerWrapper< real64 >( viewKeyStruct::defaultC11 )->
```

Comment:
I know we're inconsistent about this, but I would prefer to have almost all sub-views and sub-groups be members, and then register them with the repository. This way we do less lookup and it makes debugging substantially easier. There are cases (such as below) where you operate on the `Wrapper` instead of the wrapped object.

```c++
  this->getWrapper< array1d< real64 > >( viewKeyStruct::c66 )->
    setApplyDefaultValue( c66Default );
    setApplyDefaultValue( c66 );
```

So maybe we should instead of having the wrapped object (`array1d< real64 >`) as a member, instead have the `Wrapper< array1d< real64 > >`. This would require adding a `Group::registerWrapper( KEY_TYPE const &, WrapperBase * )` overload but it should be pretty straightforward. This would allow for the maximum debuggability.

 @rrsettgast thoughts?